src/style.css

```diff
@@ -0,0 +1,39 @@
+* {
+  margin: 0;
+}
+
+body {
+  padding: 32px 22px ;
+}
+
+.stars {
+  display: flex;
+  align-content: space-between;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+
+}
```

Comment:
```suggestion
  margin-right: 4px;
}
```
I'm not sure that last stars should have this margin too
![image](https://user-images.githubusercontent.com/96409563/221522696-50c40020-0b67-459c-ad20-b1ba03c9d040.png)